.appveyor.yml

```diff
@@ -55,19 +56,15 @@ install:
   # For building, use a new environment
   - conda create -q -n test-environment python=%PYTHON_VERSION% tk
   - activate test-environment
+  # pull pywin32 from conda because on py38 there is something wrong with finding
```

Comment:
Is that because of wrong conda above?

Reply:
I don't think so, we are still ending up with a py38 env and it should be de-coupled from Python version of the "base" conda enivorment.  I think this has to do with Python tightening up its rules about how dlls are discovered.  There are a bunch of related issues on the pywin32 issues tracker.